test/integration/webhooks/trainjob_test.go

```diff
@@ -80,7 +213,35 @@ var _ = ginkgo.Describe("TrainJob marker validations and defaulting", ginkgo.Ord
 			},
 		}
 		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
+		baseRuntimeWrapper := testingutil.MakeTrainingRuntimeWrapper(ns.Name, "testing")
+		baseClusterRuntimeWrapper := testingutil.MakeClusterTrainingRuntimeWrapper("testing")
+		trainingRuntime := baseRuntimeWrapper.RuntimeSpec(
```

Comment:
```suggestion
		trainingRuntime = baseRuntimeWrapper.RuntimeSpec(
```

Store this to remove this later.